ext/mini_racer_extension/serde.c

```diff
@@ -569,7 +577,7 @@ static int des1(char (*err)[64], const uint8_t **p, const uint8_t *pe,
             return bail(err, "negative zero bigint");
         if (pe-*p < (int64_t)u)
             goto too_short;
-        des_bigint(arg, *p, u, 1-2*t);
+        des_bigint(arg, *p, u, (int) (1-2*t));
```

Comment:
The parameter only designates the sign here: `1` or `-1`, but the type of `t` is `uint64_t`. Thus, instead of making the parameter of `des_bigint` unnecessarily wide, it's easy enough to just cast it down explicitly to avoid any potential compiler warnings that can turn into errors.